src/java/main/org/apache/zookeeper/server/command/FourLetterCommands.java

```diff
@@ -216,6 +216,10 @@ public static boolean isEnabled(String command) {
                     whiteListedCommands.add(cmd.trim());
                 }
             }
+            // It is sad that isro and srvr are used by ZooKeeper itself. Need fix this
+            // before deprecating 4lw.
+            whiteListedCommands.add("isro");
```

Comment:
@rakeshadr thanks for feedback, updated patch. I also did an optimization that only conditionally enables "isro" only when read only mode is enabled (readonly mode is disabled by default.) with a test case, so we don't have to say that "isro" is also enabled by default in doc (the less implementation details we mention there the better imo.).